_posts/en/newsletters/2025-12-19-newsletter.md

```diff
@@ -0,0 +1,1292 @@
+  for Lightning nodes to charge two additional types of fees for forwarding payments,
+  an upfront fee and a hold fee. The ultimate spender would pay the former to
+  compensate forwarding nodes for temporarily using an [HTLC][topic htlc] slot,
+  while the latter would be paid by any node that delays
+  the settlement of an HTLC, with the payment amount scaling up with the length
+  of the delay.
+
+## April
+
+{:#swiftsync}
+- **SwiftSync speedup for initial block download:** Sebastian Falbesoner
+  [posted][news349 swiftsync] to Delving Bitcoin a sample implementation
+  and results of a >5x speedup of _initial block download_ (IBD) through
+  SwiftSync, an idea initially [proposed][swiftsync ruben gh] by Ruben Somsen.
+
+  The speedup is achieved during IBD by only adding coins to the UTXO set when
+  they will still be in the UTXO set at the end of IBD. This knowledge of the
+  final UTXO set state is compactly encoded in a minimally trusted,
+  pre-generated hints file. In addition to minimizing overhead on chainstate
+  operations, SwiftSync enables further performance improvements by allowing
+  parallel block validation.
+
+  Work on a Rust implementation was [announced][swiftsync rust impl] in
+  September.
+
+{:#dahlias}
+- **DahLIAS interactive aggregate signatures:** In April, Jonas Nick, Tim Ruffing,
+  and Yannick Seurin [announced][news351 dahlias] to the Bitcoin-Dev mailing list
+  their [DahLIAS paper][dahlias paper], the first interactive
+  64-byte aggregate signature scheme compatible with the cryptographic
+  primitives already used in Bitcoin. Aggregate signatures are
+  the cryptographic requirement for [cross-input signature aggregation][topic
+  cisa] (CISA), a feature proposed for Bitcoin that could reduce the size of
+  transactions with multiple inputs, thus reducing the cost of many different
+  types of spending, [coinjoins][topic coinjoin] and [payjoins][topic payjoin]
+  included.
+
+<div markdown="1" class="callout" id="quantum">
+
+## Summary 2025: Quantum
+
+  With the increased attention on the potential for a future [quantum][topic
+  quantum resistance] computer to weaken or break the Elliptic Curve Discrete
+  Logarithm (ECDL) hardness assumption that Bitcoin relies on to prove the
+  ownership of coins, several conversations and proposals were put forward
+  throughout the year to discuss and mitigate the impact of such a development.
+
+  Clara Shikhelman and Anthony Milton [published][news357 quantum report] a
+  paper covering the impacts of quantum computing on Bitcoin and outlining
+  potential mitigation strategies.
+
+  [BIP360][] was [updated][news bip360 update] and received its BIP
+  number. This proposal has
+  garnered support both as a first step toward quantum-hardening
+  Bitcoin and an optimization for taproot use cases that do not require an
+  internal key. [Research][news365 quantum taproot] later in the year confirmed
```

Comment:
My understanding is that BIP 360 is being rewritten again, so it seems a bit premature to describe it as having garnered support, when it’s not even merged yet, because the initial draft is still in flux. See: https://github.com/bitcoin/bips/pull/1670#discussion_r2594039880

Maybe “drew interest”?